.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-pnpm lint
+pnpx --yes lint-staged
```

Comment:
issue (non-blocking): `pnpx` is deprecated, use `pnpm dlx` instead. But better yet, why not install it as a dev dependency and use `pnpm exec`? It's faster than trying to fetch it, even if from cache.

Reply:
I also wondered why the installation script (`npx mrm@2 lint-staged` from the docs) used `npx` instead of installing it, but I left it as it is (only changed `npx` to `pnpx`)